kubejs/server_scripts/monicoins/monicoins.js

```diff
@@ -1,13 +1,14 @@
-ServerEvents.recipes(event => {
```

Comment:
Pretty sure this line shouldn't be removed

Reply:
i think so too, but if you really wanted to remove it properly you'd need to remove the matching parenthesis at the end of the file